receiver/couchdbreceiver/documentation.md

```diff
@@ -30,7 +30,6 @@ metrics:
 
 | Name | Description |
 | ---- | ----------- |
-| couchdb.node.name | The name of the node. |
```

Comment:
It looks like the documentation generator needs updating here.

Reply:
Yes, this is because documentation generator doesn't get resource attributes yet. I have in the list of follow-up PRs, but I can add it to this one if you think it's needed

Reply:
Added docs generation in this PR